{{cookiecutter.slug}}/backend/{{cookiecutter.slug}}/i18n.py

```diff
@@ -0,0 +1,22 @@
+from rest_framework.decorators import api_view
+from rest_framework.request import Request
+from rest_framework.response import Response
+from django.conf import settings
+from django.utils import translation
+
+
+@api_view(['GET'])
+def get(request: Request):
+    return Response({
+        'current': request.LANGUAGE_CODE,
+        'supported': settings.LANGUAGES
+    })
+
+
+@api_view(['POST'])
+def set(request: Request):
```

Comment:
Could combine this into a single endpoint with GET and POST?
Not that important, but more RESTFUL.